app/views/WhatYouOwe.scala.html

```diff
@@ -375,6 +393,22 @@
             @messages("whatYouOwe.payments-made-bullet-1.3")</li>
         <li> @messages("whatYouOwe.payments-made-bullet-2")</li>
     </ul>
+
+    @if(creditAndRefundEnabled && whatYouOweChargesList.balanceDetails.availableCredit.isDefined && whatYouOweChargesList.balanceDetails.availableCredit.get > 0) {
```

Comment:
might be nicer like this:
whatYouOweChargesList.balanceDetails.availableCredit.map( value => {
    if(value > 0) {
        ...something
    }
}